packages/livekit-rtc/src/track.ts

```diff
@@ -45,6 +45,10 @@ export abstract class Track {
   get muted(): boolean | undefined {
     return this.info?.muted;
   }
+
+  async close() {
```

Comment:
should we have a optional boolean here to also close the source when closing the track? e.g. 
`async close(closeSource = true) {`